src/vs/workbench/contrib/chat/browser/chatListRenderer.ts

```diff
@@ -289,14 +289,6 @@ export class ChatListItemRenderer extends Disposable implements ITreeRenderer<Ch
 		this.viewModel = viewModel;
 		this._announcedToolProgressKeys.clear();
 
-		for (const templateData of this.templateDataByRequestId.values()) {
```

Comment:
with background sessions, we want things to continue in the background, so we never want to prematurely end any thinking parts just because of a context switch